charts/istio-operator/templates/istiooperator.yaml

```
@@ -4,6 +4,10 @@ kind: IstioOperator
 metadata:
   namespace: {{ .Values.istioNamespace }}
   name: istio-operator
+  annotations:
+    {{- if .Values.controlPlane.annotations }}
+    {{- toYaml .Values.controlPlane.annotations | nindent 4 }}
+    {{- end }}
```

```suggestion
  {{- with .Values.controlPlane.annotations }}
  annotations:
    {{- toYaml . | nindent 4 }}
  {{- end }}
```

We only want to add annotations if there are any being passed in.